### PR TITLE
ohio personal exemption test

### PR DESCRIPTION
**Pull request by xiaodan99**
ohio personal exemption test

### CODE REVIEW

File: policyengine_us/parameters/gov/states/oh/tax/income/oh_personal_exemption.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - threshold: 
+      2022-01-01: 40_001
+    amount:
+      2022-01-01: 2_150
+  - threshold:
+      2022-01-01: 80_000
+    amount:
+      2022-01-01: 1_900
+metadata:
+  threshold unit: currency-USD
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/oh/tax/income/oh_personal_exemption.yaml:16]:
```suggestion
  threshold_unit: currency-USD
```

File: policyengine_us/parameters/gov/states/oh/tax/income/oh_personal_exemption.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  amount unit: currency-USD
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/oh/tax/income/oh_personal_exemption.yaml:17]:
```suggestion
  amount_unit: currency-USD
```

File: policyengine_us/parameters/gov/states/oh/tax/income/oh_personal_exemption.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  period: year
+  name: oh personal exemption
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/oh/tax/income/oh_personal_exemption.yaml:19]: No longer need `name` attribute

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+class oh_senior_citizen_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio senior citizen credits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=20",
+        "https://codes.ohio.gov/ohio-revised-code/section-5747.055",
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py:12]: This links to the retirement income credit not senior citizen credit

File: policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        ).gov.states.oh.tax.income.senior_citizen.agi_cap
+        credit_amount = parameters(
+            period
+        ).gov.states.oh.tax.income.senior_citizen.credit_amount
+        has_not_taken_lump_sum_distribution = person(
+            "oh_has_not_taken_oh_lump_sum_credits", period
+        )
+
+        age = person("age", period)
+        any_over_65 = sum(age > age_threshold) >= 1
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py:32]: Tax form says it's >=

Avoid including parameter values in the code
```suggestion
        any_elderly = tax_unit.any(age >= age_threshold)
```

File: policyengine_us/variables/gov/states/oh/tax/income/is_dependent.py
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/is_dependent.py:1]: We already have central variables for this and is_spouse you can use

File: policyengine_us/variables/gov/states/oh/tax/income/oh_personal_exemption.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    reference = "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=14"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        #test for this three
+        is_dependent = person("is_tax_unit_dependent", period)#should we create a variable for this in person file? tmr ask nick
+        is_tax_unit_head = person("is_tax_unit_head", period) #the variable 'is_tax_unit_head' is defined for 'people', change this to person just for now
+        is_spouse = person("is_tax_unit_spouse", period)
+        agi = tax_unit("oh_agi", period)
+        num_of_dependents = tax_unit.sum(is_dependent)
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/oh_personal_exemption.py:20]: we already have `tax_unit_dependents`

(see https://policyengine.org/us/docs if you haven't)

File: policyengine_us/variables/gov/states/oh/tax/income/oh_personal_exemption.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+        personal_exemption_amount = parameters(
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/oh/tax/income/oh_personal_exemption.py:21]:
```suggestion
        p = parameters(period).gov.states.oh.tax.income
        personal_exemption_amount = p.exemption.personal.calc(agi)
```

**Comment by nmrodelo** [comment]
artifact of @xiaodan99's OH Personal Exemption process

**Pull request closed without merging**